src/status_im/chat/views/message/message.cljs

```diff
@@ -60,9 +61,16 @@
 
 (defview message-content-command
   [{:keys [content params] :as message}]
-  (letsubs [command [:get-command (:command-ref content)]]
+  (letsubs [command [:get-command (:command-ref content)]
+            network [:network-name]]
     (let [preview (:preview content)
-          {:keys [color] icon-path :icon} command]
+          {:keys [color] icon-path :icon} command
+          content-params (:params content)
+          send-network (-> content-params
+                           :network
+                           (string/split "_")
```

Comment:
Can this `(string/split "_"), first ` part be extracted into separate function (as it repeats here https://github.com/status-im/status-react/pull/4346/files#diff-79768cab0a0b590b170998dc7760f83aR44)?

Also what is `"_"`? Can it be extracted into constant or at least have some comment?

Reply:
@rasom get rid of this transformation and switch to just using plain map https://github.com/status-im/status-react/pull/4346/files#diff-ca9218fe8aefd36d593ca10867dddf56R16